src/NuGet.Core/NuGet.Common/Errors/IRestoreLogMessage.cs

```diff
@@ -8,13 +8,18 @@ namespace NuGet.Common
     public interface IRestoreLogMessage : ILogMessage, ILogFileContext
     {
         /// <summary>
-        /// Project or Package Id
+        /// Project or Package Id.
         /// </summary>
         string LibraryId { get; set; }
 
         /// <summary>
-        /// List of TargetGraphs
+        /// List of TargetGraphs.
         /// </summary>
         IReadOnlyList<string> TargetGraphs { get; set; }
+
+        /// <summary>
+        /// Bool indicating if this message needs to be logged to the inner logger.
+        /// </summary>
+        bool LogToInnerLogger { get; set; }
```

Comment:
How about AlwaysDisplayToUser?  LogToInnerLogger makes sense for the implementation, but it would be more clear if this was tied to the behavior.